accounts/management/commands/process_email_bounces.py

```diff
@@ -0,0 +1,79 @@
+#
+# Freesound is (c) MUSIC TECHNOLOGY GROUP, UNIVERSITAT POMPEU FABRA
+#
+# Freesound is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Freesound is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+# Authors:
+#     See AUTHORS file.
+#
+
+from django.core.management.base import BaseCommand
+from django.conf import settings
+from django.contrib.auth.models import User
+from django.utils.dateparse import parse_datetime
+from django.db import IntegrityError
+from accounts.models import EmailBounce
+from boto3 import client
+from json import loads
+import logging
+
+logger = logging.getLogger('console')
+
+
+class Command(BaseCommand):
+    help = 'Retrieves email bounce info from AWS SQS and updates db'
+
+    def handle(self, *args, **options):
+        sqs = client('sqs')
+        queue_url = settings.AWS_SQS_QUEUE_URL
+
+        total = 0
+        has_messages = True
+        while has_messages:
+            # Receive message from SQS queue
+            response = sqs.receive_message(
+                QueueUrl=queue_url,
+                MaxNumberOfMessages=1,
```

Comment:
Does this mean that you process 1 message at a time? What is the alternative - can you read more? Why would we want to read only 1 at a time instead of 10 or 50?

Reply:
Maximum number is 10, but from my testing there are usually duplicates present in them, that usually results in ~4 unique messages to process. I pondered making it a config variable because there are pros and cons for 1 and 10.